.gitlab-ci.yml

```diff
@@ -0,0 +1,45 @@
+image: humancellatlas/dss-build-box
+
+variables:
+  GIT_SUBMODULE_STRATEGY: normal
+  GITHUB_API: "https://api.github.com"
+
+stages:
+  - test
+  - deploy
+
+before_script:
+  - virtualenv ~/venv
+  - source ~/venv/bin/activate
+  - pip install -r requirements.txt
+  - source environment
+
+.tests:
+  stage: test
+  except:
+    - tags
+    - schedules
+
+security_checks:
+  extends: .tests
+  before_script:
+      - virtualenv ~/venv
+      - source ~/venv/bin/activate
+  script:
+    - pip install trufflehog
+    - wget -O regex.json https://raw.githubusercontent.com/HumanCellAtlas/dcplib/master/components/trufflehog_regex_patterns.json
+    - trufflehog --regex --rules regex.json --entropy=False https://github.com/HumanCellAtlas/fusillade.git
+    - rm regex.json
+
+deploy:
+  stage: deploy
+  script:
+    - ./scripts/setup_fusillade.py $CI_COMMIT_REF_NAME --file ./config/groups.json -d
```

Comment:
what is $CI_COMMIT_REF_NAME here - the branch name (and therefore the stage to deploy to)?